packages/model-viewer/src/features/annotation.ts

```diff
@@ -194,6 +205,41 @@ export const AnnotationMixin = <T extends Constructor<ModelViewerElementBase>>(
       hotspot.updateNormal(config.normal);
     }
 
+    /**
+     * This method returns the world position and normal of the point on the
+     * mesh corresponding to the input pixel coordinates given relative to the
+     * model-viewer element. The position and normal are returned as strings in
+     * the format suitable for putting in a hotspot's data-position and
+     * data-normal attributes. If the mesh is not hit, position returns the
+     * empty string.
+     */
+    getHitResult(pixelX: number, pixelY: number): HitResult {
```

Comment:
It seems like a `HitResult` is a descriptor for a plane. It might help folks to understand the significance of `normal` (which our users have reported is confusing at first) if we use language that is more natural to what the API does for them. e.g., `planeFromPoint` or `surfaceFromPoint` or something to that effect. Suggested naming inspired by [`elementFromPoint`](https://developer.mozilla.org/en-US/docs/Web/API/DocumentOrShadowRoot/elementFromPoint).

Also: it would be great to see some tests for this 👍